packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -19,6 +20,7 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
   const state = useMemo(() => ({ patient, patientUuid }), [patient, patientUuid]);
+  const { active } = useWorkspace();
```

Comment:
Yeah generally I don't like using this hook multiple times.

Maybe we can refactor such that only the patient chart component uses it and it forwards the elements to the workspace component. Not sure yet.

We can leave as is for now.